okta_openvpn.py

```diff
@@ -40,18 +40,18 @@
                   python_version=platform.python_version())
 log = logging.getLogger('okta_openvpn')
 log.setLevel(logging.DEBUG)
-syslog = logging.handlers.SysLogHandler()
-syslog_fmt = "%(module)s-%(processName)s[%(process)d]: %(name)s: %(message)s"
-syslog.setFormatter(logging.Formatter(syslog_fmt))
-log.addHandler(syslog)
+#syslog = logging.handlers.SysLogHandler()
+#syslog_fmt = "%(module)s-%(processName)s[%(process)d]: %(name)s: %(message)s"
+#syslog.setFormatter(logging.Formatter(syslog_fmt))
+#log.addHandler(syslog)
 # # Uncomment to enable logging to STDERR
-# errlog = logging.StreamHandler()
```

Comment:
There's no syslog in docker container, so it would be better to log it to stdout or stderr. This way log lines should be visible using `docker logs`